source/common/json/config_schemas.cc

```diff
@@ -441,7 +441,7 @@ const std::string Json::Schema::REDIS_PROXY_NETWORK_FILTER_SCHEMA(R"EOF(
       "conn_pool" : {"type" : "object"}
     },
     "required": ["cluster_name", "stat_prefix", "conn_pool"],
-    "additionalProperties": false
+    "additionalProperties": true
```

Comment:
We shouldn't be making any v1 changes anymore so we should be able to revert this change.

Reply:
ok .. I'll change the proxy_filter_test.cc code to add config that is not v1 to avoid the schema validation exception.